assets/src/domain/eventEditor/services/apollo/mutations/tickets/useOnCreateTicket.ts

```diff
@@ -22,38 +22,43 @@ const useOnCreateTicket = (): TicketMutationCallbackFn => {
 				// to avoid refetching of prices.
 				updatePriceCache({ proxy, prices, ticketIn, ticketId, action: 'add' });
 
-				// Set relations with datetimes
-				updateRelations({
-					entity: 'tickets',
-					entityId: ticketId,
-					relation: 'datetimes',
-					relationIds: datetimeIds,
-				});
-				datetimeIds.forEach((entityId: string) => {
-					addRelation({
-						entity: 'datetimes',
-						entityId,
-						relation: 'tickets',
-						relationId: ticketId,
+				// if related datetimes are passed
+				if (datetimeIds?.length) {
+					// Set relations with datetimes
+					updateRelations({
+						entity: 'tickets',
+						entityId: ticketId,
+						relation: 'datetimes',
+						relationIds: datetimeIds,
 					});
-				});
+					datetimeIds.forEach((entityId: string) => {
+						addRelation({
```

Comment:
notice here that there are 2 actions pertaining to relations, later we might consider consolidating these together in a separate hook, maybe